core/domain/stats_services.py

```diff
@@ -1198,10 +1197,9 @@ def record_learner_answer_info(
             value will be equal to 'exp_id:state_name' and for question
             this will be equal to 'question_id'.
         interaction_id: str. The ID of the interaction.
-        answer: dict or list or str or int or bool. The answer which is
-            submitted by the learner. Actually type of the answer is
-            interaction dependent, like TextInput interactions have
-            string type answer, NumericInput have int type answers etc.
+        answer: *(json-like). The answer which is submitted by the
```

Comment:
Add a space after *

Reply:
Adding space after * results in lint error of missing-parameter-documentation. So I left it as it is

Reply:
Done